### DISCUSSION TITLE
After installation I've lost access to Start Menu

### DISCUSSION THREAD

**Discussion by HakooDesai** [valinet/ExplorerPatcher#2116] [Q&A]
I click on Start button, but it's not opening anything no matter how many times I click or how long I wait.

This is happening for right click on pinned shortcut as well, but at least can access context menu after sometime.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

And check if symbols were downloaded: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

You can check using the console in explorer patcher.  Open up the properties application for explorer patcher, go to advanced, and click on enable console.

to run the explorer patcher properties application, run this from the run menu:
C:\Windows\System32\rundll32.exe "C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll",ZZGUI
